src/ActionList.jsx

```diff
@@ -1,5 +1,6 @@
 import React, { Component } from 'react';
 import ReactDOM from 'react-dom';
+import dragula from 'react-dragula';
```

Comment:
By having it separated, you mean the same? To have `ActionList` and `ActionListDraggable` components?